### PR TITLE
Update SDK + other fixes

### PR DESCRIPTION
**Pull request by juanjux**
- Upgrade to SDK 2.14 and annotate or drop (with "TODO"s) the
      nodes failing the new check.
    - Fix position annotation.
    - Other microfixes.
    
    Signed-off-by: Juanjo Alvarez <juanjo@sourced.tech>

### CODE REVIEW

File: driver/normalizer/normalizer.go [hunk 1]
```diff
@@ -80,17 +80,32 @@ var Normalizers []Mapping = []Mapping{
 
 	// iflipflop / eflipflop, have selector but not names
 	MapSemantic("flip_1", uast.Identifier{}, MapObj(
-		Obj{uast.KeyToken: Var("ident")},
+		Obj{
+			uast.KeyToken: Var("ident"),
+			// could not find any case where this wasn't nil, but this will
+			// detect if we find any
+			"base": Is(nil),
+			"qnames": Any(), // same as ident
```
Comment (dennwc) [inline driver/normalizer/normalizer.go]: Then:
```suggestion
			"qnames": Var("ident"),
```
Same below.

Comment (juanjux) [reply driver/normalizer/normalizer.go]: Same but with slightly different format so using the same Var fail the validation.

Comment (dennwc) [reply driver/normalizer/normalizer.go]: Ah, okay, then it's fine. Just a short comment regarding the difference will be nice.

File: native/lib/ruby_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require_relative './ruby_driver/node_converter'
```
Comment (creachadair) [inline native/lib/ruby_test.rb]: It seems unfortunate that we have to manually write this out for each test case. :(

File: driver/normalizer/normalizer.go (hunk 1 above)
Comment (juanjux) [reply driver/normalizer/normalizer.go]: That makes sense, yes.

File: native/lib/ruby_test.rb (hunk 2 above)
Comment (juanjux) [reply native/lib/ruby_test.rb]: Maybe there is a better way. I basically learned Ruby for this driver so I guess it'll be full of non-idiomatic stuff.

**Pull request merged**